scripts/lib/gh.sh

```diff
@@ -1,113 +0,0 @@
-#!/usr/bin/env bash
-
```

Comment:
substitute for this file?(used in auto-generate-controllers prow job)

Reply:
Should this be moved to the code-generator `scripts` directory instead?

Reply:
It is mainly used by the CD process(auto-generating controllers). I think it should stay in test-infra.

Reply:
Moved to `cd/lib` and anything using it has been updating accordingly